tests/test_handler_lr_scheduler.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2020 MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import torch
+import unittest
+import numpy as np
+from ignite.engine import Engine, Events
+from monai.networks.nets import UNet
+from monai.handlers import LrScheduleHandler
+import logging
+import sys
+
+
+class TestHandlerLrSchedule(unittest.TestCase):
+    def test_content(self):
+        logging.basicConfig(stream=sys.stdout, level=logging.INFO)
+        data = [0] * 8
+
+        # set up engine
+        def _train_func(engine, batch):
+            pass
+
+        val_engine = Engine(_train_func)
+        train_engine = Engine(_train_func)
+
+        @train_engine.on(Events.EPOCH_COMPLETED)
+        def run_validation(engine):
+            val_engine.run(data)
+            val_engine.state.metrics["val_loss"] = 1
+
+        # set up testing handler
+        net = UNet(
```

Comment:
If you wanted to simplify things `net` could just be a single nn.Module object like conv2d if you just need something to pass to the optimizer.

Reply:
updated to use `torch.nn.PReLU()` in the latest patch.
Thanks.